prime-router/src/main/kotlin/cli/tests/SftpcheckTest.kt

```diff
@@ -0,0 +1,120 @@
+package gov.cdc.prime.router.cli.tests
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory
+import com.fasterxml.jackson.module.kotlin.KotlinModule
+import com.fasterxml.jackson.module.kotlin.readValue
+import gov.cdc.prime.router.DeepOrganization
+import gov.cdc.prime.router.common.Environment
+import gov.cdc.prime.router.cli.SettingCommand
+import gov.cdc.prime.router.cli.SettingsUtilities
+import org.apache.http.HttpStatus
+import java.io.File
+
+/**
+ * Test SFTP receiver connections.  It checks the ignore.XYZ organization for the connection:
+ *  For each sftp receiver listed in the ignore organization, call the sftpcheck API
+ *  endpoint and verify you get back an OK response that we are able to connect to
+ *  the SFTP server.
+ *  This test needs to be able to run in staging and local, but not production.
+ *  When the smoke tests are running in sequential mode then have this new test
+ *  run after 'ping' and before 'end2end'.
+ *
+ * @returns:
+ *  If SUCCESS:
+ *      good("Test passed: SFTP")
+ *  If ERROR:
+ *      bad("Test SFTP receiver connections: Failed"
+ *
+ */
+
+class SftpcheckTest : CoolTest() {
+    override val name = "sftpcheck"
+    override val description = "Test SFTP receiver connections"
+    override val status = TestStatus.DRAFT
+
+    /**
+     * Define private local variables for use in the test.
+     */
+    private val receiverOrganizationPath = "./settings/organizations.yml"
+    private val dummyAccessToken = "dummy"
+    private val sftpcheckUri = "/api/check?sftpcheck="
+    private val sftpcheckMessage = "Test SFTP receiver connections: "
+    var testResult = true
+    val yamlMapper: ObjectMapper = ObjectMapper(YAMLFactory()).registerModule(KotlinModule())
+    val allowEnvironment = arrayListOf(Environment.LOCAL, Environment.STAGING)
+
+    override suspend fun run(environment: Environment, options: CoolTestOptions): Boolean {
+
+        /**
+         * Check for allowable environment to run the test.
+         */
+        if (!allowEnvironment.contains(environment))
```

Comment:
Check other CLI comments for the env option instead of checking for the environment directly that we can run from local to other environments.

Reply:
This line will be removed.  According to the environment (env) set of the TestReportStream.kt lines 131 - 141, it will allow only "test", "local", "staging" to run any test.  It prevents test to run under "prod" environment at line 140. 
![image](https://user-images.githubusercontent.com/20068283/141203226-f5582a83-4504-41bf-bd68-c3651ea1155f.png)